tests/API/HcaApiTestAutomationFramework/HcaApiTestAutomationFramework/GraphQlDTO/ClientConfigurationRequestDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace HcaApiTestAutomationFramework.GraphQlDTO
+{
+	public class ClientConfigurationRequestDTO
+	{
+		public Clientsdkmetadata clientSdkMetadata { get; set; }
+		public string query { get; set; }
+		public string operationName { get; set; }
```

Comment:
The names of all public fields, properties must begin with a capital letter
```suggestion
		public ClientSdkMetadata СlientSdkMetadata { get; set; }
		public string Query { get; set; }
		public string OperationName { get; set; }
```